lib/ClangImporter/ClangModuleDependencyScanner.cpp

```diff
@@ -348,18 +353,14 @@ bool ClangImporter::addBridgingHeaderDependencies(
   // Retrieve or create the shared state.
   auto clangImpl = getOrCreateClangImpl(cache);
 
-  // Reform the Clang importer options.
-  // FIXME: Just save a reference or copy so we can get this back.
-  ClangImporterOptions importerOpts;
-
```

Comment:
hmm, is this plumbing necessary if we add here `importerOpts.ExtraArgs = getExtraClangArgs();`?

Reply:
🤦‍♂️ Can't believe I missed that. 
Good catch, thank you!